Add documentation for `@apollo/generate-persisted-query-manifest` to its README (#314)
apollographql/apollo-utils: packages/generate-persisted-query-manifest/README.md

# `@apollo/generate-persisted-query-manifest`

## Setup

First install the package that will extract queries from your app

```sh
npm install --save-dev @apollo/generate-persisted-query-manifest
```

Run code extraction using the CLI

```sh
npx generate-persisted-query-manifest
```

## CLI configuration

> If you need to provide further customization, such as customizing the location of where to search for your documents, you can provide a config file at the root of your project. Create a `persisted-query-manifest.config.json` file in the root of your project.

Reviewer: ```suggestion
To override the default options, you can provide a config file. Create a `persisted-query-manifest.config.json` file in the root of your project.
```
